code/processes/housekeeping.q

```diff
@@ -91,6 +94,11 @@ rm:{[FILE]
 //-zips files
 zip:{[FILE]
 	@[{.lg.o[`housekeeping;"zipping ",x]; system "gzip ",x};FILE; {.lg.e[`housekeeping;"Failed to zip ",x," : ", y]}[FILE]]}
+
+//-creates a tar ball from a directory and removes original directory and files
+tardir:{[DIR]
+	@[{.lg.o[`housekeeping;"creating tar ball from ",x]; system "tar -czf ",x,".tar.gz ",x," --remove-files"};DIR; {.lg.e[`housekeeping;"Failed to create tar ball from ",x," : ", y]}[DIR]]}
```

Comment:
Could you put the full path for tar on here the same way we have for find

Reply:
Another place to use `.proc.sys` when it exists.